app/views/docs/authentication-management.phtml

```diff
@@ -0,0 +1,397 @@
+<p>
+    Appwrite has built-in features to help manage user accounts.
+    Users can be organized into teams and be given labels, so they can be given different permissions and access different resources.
+    Each user can also have their own preference object, which you can use to save preferences such as theme, language, and notification settings.
+</p>
+
+
+<h2><a href="#preferences" id="preferences">User Preferences</a></h2>
+<p>
+    You can store user preferences on a user's account using Appwrite's <a href="/docs/client/account#accountUpdatePrefs">Update Preferences</a> endpoint. 
+    You can store user preferences such as theme, notification settings, or preferred language so they can be synced across multiple devices.
+</p>
+<p>
+    Preferences are stored as a key-value JSON object.
+    The maximum allowed size for preferences is 64kB, and an error will be thrown if this limit is exceeded.
+</p> 
+<ul class="phases clear" data-ui-phases>
+<li>
+        <h3>Web</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client()
+    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('[PROJECT_ID]');               // Your project ID
+
+const account = new Account(client);
+
+const user = await account.updatePrefs({darkTheme: true, language: 'en'});</code></pre>
```

Comment:
Lets use typical promise.then() syntax

Reply:
Please check the whole PR. This should be done in ~dozen palces.

Reply:
Some places, we can't use promise.then(), basically when ever we have a 2 part example where you need to save something from the returned object.

But this should indeed be .then

Reply:
Fixed here. There are some palces where this is used on purpose, because we're saving the objects returned.